README.md

```diff
@@ -288,26 +273,20 @@ Click [here](https://s3-eu-west-1.amazonaws.com/ih-materials/uploads/upload_00ab
 
 <br>
 
-
-
 ## Test Your Code
 
 We will be working with automated tests again!
 
-Please, open your terminal, change directories into the root of the lab, and run `npm install` to install the test runner. Now, you can run the `npm run test:watch` command to run automated tests in *watch mode*. Open the resulting `lab-solution.html` file using the "Live Server" VSCode extension to always see the test results.
+Please, open your terminal, change directories into the root of the lab, and run `npm install` to install the test runner. Now, you can run the `npm run test:watch` command to run automated tests in _watch mode_. Open the resulting `lab-solution.html` file using the "Live Server" VSCode extension to always see the test results.
```

Comment:
why did you change the * to a _ ? auto formatter? 
Pay attention to what you are commiting - stuff like this has caused bugs in companies ^^ 
also for a reviewer it's harder to see, what you actually changed. i.e. the important stuff.